source/quickinstallationguide/qig.rst

```diff
@@ -86,10 +86,10 @@ CloudStack. We will go over the steps to prepare now.
 Operating System
 ~~~~~~~~~~~~~~~~
 
-Using the CentOS 7.9.2009 minmal x86_64 install ISO, you'll need to install
+Using the CentOS 7.9.2009 minimal x86_64 install ISO, you'll need to install
 CentOS 7 on your hardware. The defaults will generally be acceptable for this
 installation - but make sure to configure IP address/parameters so that you can later install needed
-packages from internet. Later, we will change the network configuration as needed.
+packages from the internet. Later, we will change the network configuration as needed.
```

Comment:
I am not an English speaker but isn't it either `internet` or `the Internet` ? (note the capitalisation)

Reply:
@bradh, so capitalisation is optional if the article is used?

Reply:
Nouns (other than pronouns) are not usually capitalised. 

Seriously though, its not a big deal. The only thing I was trying to do in this PR was fix the spelling mistake. I can back out the rest of it.

Reply:
no, don´t bother I'll merge